### DISCUSSION TITLE
Any plans for Anthropic Prompt Caching?

### DISCUSSION THREAD

**Discussion by TIAcode** [frankroeder/parrot.nvim#50] [Ideas]
Great plugin!

I think a lot of users of this often have quite a lot of context and often ask follow up questions, so some savings could be had from using Prompt Caching (where cached message cost like 10% per token). Also it looks like the `ephemeral` caching doesn't really need that much / any state saved (I could be wrong), though obviously it would be nice if we could have some big context prompts cached over sessions, but I'm not really sure how that would work UI-wise.

**Comment by frankroeder** [comment]
Hi @TIAcode,
I'm glad you like this fork! For the chat windows, it could simply be a line like `system_cache: <your text>`. For interactive elements, it might be more complex. Let me think about it, and I might have something next week. Thank you very much for the suggestion!

**Reply by frankroeder** [reply]
Dear @TIAcode , the current approaches that I came up with are too hacky and overly specific to the Anthropic API. I apologize that I cannot provide a quick solution for integration at this time.
